src/main/java/com/parksupark/soomjae/server/member/util/RandomNicknameCreator.java

```diff
@@ -0,0 +1,16 @@
+package com.parksupark.soomjae.server.member.util;
+
+import java.security.SecureRandom;
+
+public class RandomNicknameCreator {
+
+    // SecureRandom 인스턴스는 생성 비용이 비싸므로 재사용
+    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
+
+    public static String createRandomNickname() {
+        int randomSuffix = SECURE_RANDOM.nextInt(900000) + 100000;
```

Comment:
```suggestion
        int randomSuffix = SECURE_RANDOM.nextInt(100_000, 1_000_000);
```

개인적으로는 위의 형태가 더 읽기 편한 것 같습니다.